### PR TITLE
add propagators for lightstep tracer

### PR DESCRIPTION
**Pull request by vetinari**
this enables skipper to pick up traces in B3 format when enabled.

With a grafana which can propagate in B3 format the traces can be continued (when both
skipper and grafana's jaeger collector report to the same satellites).

Also: updates lightstep tracer from 0.16.0 to 0.22.0

Signed-off-by: Hanno Hecker <hanno@zalando.de>

### CODE REVIEW

File: tracing/tracers/lightstep/lightstep.go [hunk 1]
```diff
@@ -101,6 +106,22 @@ func parseOptions(opts []string) (lightstep.Options, error) {
 			if maxBufferedSpans, err = strconv.Atoi(parts[1]); err != nil {
 				return lightstep.Options{}, fmt.Errorf("failed to parse max buffered spans: %v", err)
 			}
+		case "propagators":
+			if len(parts) > 0 {
```
Comment (AlexanderYastrebov) [inline tracing/tracers/lightstep/lightstep.go:110]: > 1

Comment (vetinari) [reply tracing/tracers/lightstep/lightstep.go]: yes, fixed

**Comment by aryszka** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Pull request merged**